db/schema.rb

```diff
@@ -90,10 +90,10 @@
     t.string   "processing_state",        default: "requested", null: false
     t.datetime "created_at",                                    null: false
     t.datetime "updated_at",                                    null: false
-    t.string   "reference_no"
-    t.boolean  "closed"
     t.boolean  "override_delivery_error", default: false
     t.string   "delivery_error_type"
+    t.string   "reference_no"
+    t.boolean  "closed"
```

Comment:
I know.  It's something that pesters me all the time and I'm afraid I let it slip past this time.